projects/oak/Dockerfile

```diff
@@ -16,13 +16,21 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder
 MAINTAINER tzn@google.com
-RUN apt-get update && apt-get install -y \
-  curl
+
+# Use a fixed Bazel version.
+ARG bazel_version=1.1.0
```

Comment:
Could you add a link to the Asylo dockerfile, so that we can easily compare if they are in sync?

Reply:
Done